exporter/prometheusremotewriteexporter/config.go

```diff
@@ -124,5 +124,9 @@ func (cfg *Config) Validate() error {
 		cfg.MaxBatchSizeBytes = 3000000
 	}
 
+	if len(cfg.ClientConfig.Compression) > 0 && cfg.ClientConfig.Compression != "snappy" {
```

Comment:
CompressionType is a string[[1](https://github.com/open-telemetry/opentelemetry-collector/blob/b741c486d90f6e20c429807519c31d6ef4659f52/config/configcompression/compressiontype.go#L12)], there's no need to check for length :) 

```suggestion
	if cfg.ClientConfig.Compression != "snappy" {
```

Reply:
Thanks for the review. `len(cfg.ClientConfig.Compression) > 0` is required as the `Compression` field [by default is omitted](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confighttp/confighttp.go#L74) if not set by the user explicitly and would cause the validation to fails as the suggested change will always expect the filed to be specified.

Reply:
Thanks for the review. `len(cfg.ClientConfig.Compression) > 0` is required as the `Compression` field is omitted [[1](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configgrpc/configgrpc.go#L72), [ 2](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confighttp/confighttp.go#L74)] if not set by the user explicitly and would cause the validation to fail.

Reply:
@ArthurSens Are we happy to move forward with the current state?

Reply:
I see! Briliant, thank you